Sources/XcodeProj/Extensions/String+md5.swift

```diff
@@ -21,28 +21,29 @@
  */
 
 import Foundation
+import CommonCrypto
 
 extension String {
-    var md5: String {
-        if let data = data(using: .utf8, allowLossyConversion: true) {
-            let message = data.withUnsafeBytes { bufferPointer in
-                Array(UnsafeBufferPointer(
-                    start: bufferPointer.baseAddress?.assumingMemoryBound(to: UInt8.self),
-                    count: data.count
-                ))
-            }
-
-            let MD5Calculator = MD5(message)
-            let MD5Data = MD5Calculator.calculate()
+    private func toHex(_ char: UInt8) -> UInt8 {
+        if char < 10 {
+            return char + 48
+        } else {
+            return (char - 10) + 65
+        }
+    }
 
-            var MD5String = String()
-            for c in MD5Data {
-                MD5String += String(format: "%02x", c)
+    var md5: String {
+        guard let data = data(using: .utf8, allowLossyConversion: true) else { abort() }
```

Comment:
Aborting with no message is a terrible user experience for the user. What about passing a string with a explanatory comment of what happened?